notify/log/log_unix.go

```diff
@@ -0,0 +1,112 @@
+//go:build !windows
+
+/*
+ * Copyright (c) 2022, MegaEase
+ * All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package log
+
+import (
+	"fmt"
+	"log/syslog"
+	"net"
+	"strconv"
+	"strings"
+
+	"github.com/megaease/easeprobe/global"
+	"github.com/megaease/easeprobe/report"
+	log "github.com/sirupsen/logrus"
+)
+
+func (c *NotifyConfig) checkNetworkProtocol() error {
+	if len(c.Network) == 0 {
+		return fmt.Errorf("protocol is required")
+	}
+	if len(c.Host) == 0 {
+		return fmt.Errorf("host is required")
+	}
+	if c.Network != TCP && c.Network != UDP {
+		return fmt.Errorf("[%s] invalid protocol: %s", c.NotifyKind, c.Network)
+	}
+	_, port, err := net.SplitHostPort(c.Host)
+	if err != nil {
+		return fmt.Errorf("[%s] invalid host: %s", c.NotifyKind, c.Host)
+	}
+	_, err = strconv.Atoi(port)
+	if err != nil {
+		return fmt.Errorf("[%s] invalid port: %s", c.NotifyKind, port)
+	}
+	return nil
+}
+
+// ConfigLog configures the log
+// Unix platform support syslog and log file notification
+func (c *NotifyConfig) ConfigLog() error {
+	c.NotifyKind = "log"
+	c.NotifyFormat = report.Log
+	c.NotifySendFunc = c.Log
+
+	c.logger = log.New()
+
+	isSyslog := (strings.TrimSpace(c.File) == "syslog")
+	hasNetwork := false
+	if isSyslog == true {
+		if len(c.Network) == 0 || len(c.Host) == 0 {
+			hasNetwork = false
+		} else {
+			err := c.checkNetworkProtocol()
+			if err != nil {
+				return err
+			}
+			hasNetwork = true
+		}
+	}
+
+	// syslog && network configuration error
+	if isSyslog && hasNetwork == true { // remote syslog
+		c.NotifyKind = "syslog"
```

Comment:
`"syslog"` appears multiple times,suggest to to declare as a constant string
Refer to: `https://golangbot.com/constants/`